osu.Game/Overlays/BeatmapSet/BeatmapSetLayoutSection.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Effects;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.BeatmapSet
+{
+    public class BeatmapSetLayoutSection : Container
```

Comment:
please document in the future. it's really hard to tell why these kinds of rogue classes exist otherwise.